src/plugins/js-eval/jsEvalPlugin.js

```diff
@@ -18,31 +18,30 @@ const jsEvalPlugin = async ({ mentionUser, respond, message, selfConfig, log })
     didRespond = true;
     children.delete(child);
 
-    const msg = err && err.killed ? 'Timeout' : stdout.trim();
-    const prefix = mentionUser ? `${mentionUser}, ` : err ? '(error) ' : '(okay) ';
-    respond(prefix + msg);
+    const prefix = mentionUser ? `${mentionUser}, ` : err ? (err.killed ? '(timeout) ' : '(error) ') : '(okay) ';
+    respond(prefix + stdout.trim());
```

Comment:
@brigand Could you, in the future add this change? it allows to print all the output until the timeout occurs